app/policies/article_policy.rb

```diff
@@ -12,19 +12,47 @@ def self.limit_post_creation_to_admins?
     FeatureFlag.enabled?(:limit_post_creation_to_admins)
   end
 
+  # @note [@jeremyf] I am re-implemnenting the initialize method, but removing the Pundit
+  #       authorization.  There's an assumption that all policy questions will require a user,
+  #       unless you know specifically that they don't.
+  #
+  # @todo [@jeremyf] I don't like altering the initializer and it's core assumption.  But the other
```

Comment:
```suggestion
  # @todo [@jeremyf] I don't like altering the initializer and its core assumption.  But the other
```

Reply:
Someday, I'll get that "it's/its" correct.